README.md

```diff
@@ -9,59 +7,70 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
 ### Install
 
 ```
-npm install loopback-connector-couchbase3 --save
+npm install loopback-connector-couchbaseX --save
 ```
 
-### Config
+### DataSource Config in LoopBack
 
+- For Couchbase version >= 5, as default
 ```
 # datasources.json
 {
-  "couchbaseTestBucket": {
-    "name": "couchbaseTestBucket",
-    "connector": "couchbase3",
+  "testDs": {
+    "name": "testDs",
+    "connector": "couchbase5",
+    "version": 5,  // optional
```

Comment:
here you are making `version` to be optional by relying on the `NaN comparison` always returns false which is too subtle to understand, you can still leave `verison` optional but set the default value(e.g. 5) around https://github.com/Wiredcraft/loopback-connector-couchbase3/pull/89/files#diff-afa091a9bb8da3eaff5c46270d3ed166R74 ,